src/vivarium_csu_ltbi/data/cli.py

```diff
@@ -3,10 +3,10 @@
 
 import click
 from loguru import logger
-
 from vivarium import Artifact
 from vivarium_cluster_tools.psimulate.utilities import get_drmaa
-from vivarium_csu_ltbi.data.globals import COUNTRIES, formatted_country
+
+from vivarium_csu_ltbi import globals as ltbi_globals
```

Comment:
I also changed country to location pretty much everywhere. Some places we were using country, some location, and some country name.  In practice, we might get requests for nationals and subnationals in the same project and location is the term of art around the institute for all such things.